apps/chat/src/utils/app/file.ts

```diff
@@ -123,19 +123,19 @@ export const getFilesWithInvalidFileType = (
     ? []
     : files.filter((file) => !isAllowedMimeType(allowedFileTypes, file.type));
 };
-export const notAllowedSymbols = ':;,=/{}%&\\\t\x00-\x1F';
+export const notAllowedSymbols = ':;,=/{}%&\\';
 export const notAllowedSymbolsRegex = new RegExp(
-  `[${escapeRegExp(notAllowedSymbols)}]|(\r\n|\n|\r|\t)`,
+  `[${escapeRegExp(notAllowedSymbols)}]|(\r\n|\n|\r|\t)|[\x00-\x1F]`,
```

Comment:
I suppose this change disallow `[` and `]`, Please, use just `\x00-\x1F`. At least double check it

Reply:
https://javascript.info/regexp-character-sets-and-ranges#ranges
that's a way to exclude a range